modules/op-creating-pipeline-tasks.adoc

```diff
@@ -57,5 +57,5 @@ tkn                                      1 day ago
 
 [IMPORTANT]
 ====
-In {pipelines-title} 1.10, cluster task functionality is deprecated and is planned to be removed in a future release.
+In {pipelines-title} 1.10, `ClusterTask` functionality  is deprecated and is planned to be removed in a future release.
```

Comment:
see previous comment about backticks

Reply:
See above - CR name, also keyword used in YAML